docs/management/connectors/action-types/email.asciidoc

```diff
@@ -228,6 +228,10 @@ is considered `false`.  Typically, `port: 465` uses `secure: true`, and
 [[elasticcloud]]
 ==== Sending email from Elastic Cloud
 
+IMPORTANT: You need to 
+  link:{cloud}/ec-watcher.html#ec-watcher-whitelist[allowlist] the email 
```

Comment:
I think this URL will need to change per https://github.com/elastic/cloud/pull/103642
```suggestion
  link:{cloud}/ec-watcher.html#ec-watcher-allowlist[allowlist] the email 
```